cmd/identity/main.go

```diff
@@ -168,7 +169,9 @@ func cmdAuthorize(cmd *cobra.Command, args []string) error {
 		config.Signer.Address = defaultSignerAddress
 	}
 
-	signedChainBytes, err := config.Signer.Sign(ctx, ident, authToken)
+	verClient := version.NewClient()
+
+	signedChainBytes, err := config.Signer.Sign(ctx, ident, verClient, authToken)
```

Comment:
Passing version info the the sign function feels wrong.

Reply:
Yeah, it kind of is.
But it is using the transport client to make a GRPC Call..

Reply:
Maybe the argument should be the transport.Client instead?